### DISCUSSION TITLE
Can I not have free and pro plan under the same organization?

### DISCUSSION THREAD

**Discussion by pierroo** [supabase/supabase#20581] [Questions] [answered]
I will finally soon be going live with my project.

I have two projects in my supabase organization, one which is meant to be a hobby; the other that will go live.
I only want to enable pricing for my real project, while the other should remain free entirely, but when I wanna upgrade I have no way to select which one I want to activate for "XS compute hours"

![image](https://github.com/supabase/supabase/assets/17619412/ee0a478e-22b8-4ecc-a443-7bc9e4112e7d)

Do I really have to create a separate organization for all hobby projects?

**Comment by GaryAustin1** [comment, marked as answer]
Free plans have to be in their own organization.
